baybe/campaign.py

```diff
@@ -3,7 +3,7 @@
 from __future__ import annotations
 
 import json
-from typing import List
+from typing import List, Optional
```

Comment:
@AdrianSosic any hints as to what functioanlity od botorch has to be hanged too? Ie do we need an explicit call to some `prior` fucntion? Are the two cases no-data / data explicitly to be distinguished or are they naturally treated by one and the same call?

Reply:
Conceptually (i.e. mathematically), it should just work. But botorch is a bit weird IMO when it comes to situations without data. I've seen the same behavior when I implemented the transfer learning kernel: there actually is a botorch builtin MultiTaskGP that would do everything out of the box. However, that object just doesn't work properly when you add a "new" (i.e. completely unobserved task), even though mathematically the prior is completely defined for it and would still be helpful in that it would simply "interpolate" the other tasks.

Similar situation here. One thing is that botorch/gpytorch models complain when they are being used without having trained them / training would potentially fail with empty data. So the solution is basically to work around these blockers. Unfortunately, I have no magic solution to it and would also need to dig through the problems step by step :/ But I suspect it shouldn't be too hard, actually